etc/opensc.conf.in

```diff
@@ -301,6 +301,69 @@ app default {
 		# md_guid_as_label: Use the GUID generated for the key as label in the PKCS#15 structure (Default: false - no label set)
 		# md_supports_container_key_gen: Card allows generating key pairs on the card (Default: false)
 		# md_supports_container_key_import: Card allows importing private keys (Default: false)
+		#
+		# Window title of the PIN pad dialog
```

Comment:
Detault text and locales are part of `src/ui/strings.c`. I never used gettext and looking at the Po files, I think raw C code is easier to handle.

The configuration file should be used if the technical naming of card names, PINs and handling the card is not intuitive enough. Yes, at some point (https://github.com/OpenSC/OpenSC/pull/1126/commits/378bc1f3814c78d31a56f3dd9cb44d9fd32b7d13) I hard coded the strings you are currently seeing in the config file, but I later wanted to avoid mixing the card layer with the minidriver.

What's the problem with a big configuration file? We could easily reduce the size of the configuration file if we would remove comments. However, the content needs to be put into a manpage (and into the wiki)...